components/TextIamgeTransparent.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+    <div class="textimagetransparent">
+        <img class="image" :src="image" />
+        <div class="content">
+            <h1>Login</h1>
+            <input type="text" placeholder="Username" />
+            <input type="password" placeholder="Password" /><br />
+            <a class="link" href="https://google.com">Reset Password</a>
+
+            <div class="buttons">
+                <button class="button">Login</button>
+                <button class="button">Register</button>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+interface Props {
+    image: string;
+}
+
+defineProps<Props>();
+</script>
+<style lang="scss" scoped>
+.image {
+    width: 100%;
+    height: 100vh;
+    object-fit: cover;
+}
+
+.content {
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 40vw;
+    height: 100vh;
+    background-color: rgba($color: #ffff, $alpha: 0.5);
+    padding-left: 5%;
+    padding-top: 10%;
+}
+
+.content h1 {
+    font-size: var(--font-size-xl);
+}
+
+.content input {
+    width: 50%;
+    margin: 0.5rem 0rem;
+}
+
+.textimagetransparent {
+    position: relative;
+    text-align: left;
+    color: white;
+}
+
+.link {
```

Comment:
CSS selectors should just use the component name, this makes the classes redundant